tmva/tmva/src/MethodDL.cxx

```diff
@@ -2054,20 +2036,13 @@ std::vector<Double_t> MethodDL::GetMvaValues(Long64_t firstEvt, Long64_t lastEvt
 #endif
 
 #endif
-   } else if (this->GetArchitectureString() == "CPU") {
-//#ifdef R__HAS_TMVACPU
+   } else {
       Log() << kINFO << "Evaluate deep neural network on CPU using batches with size = " << batchSize << Endl << Endl;
       return PredictDeepNet<DNN::TCpu<ScalarImpl_t> >(firstEvt, lastEvt, batchSize, logProgress);
-//#endif
    }
-   Log() << kINFO << "ERROR:  STANDARD architecture  is not supported anymore for MethodDL ! "
-         << Endl << Endl;
-// #if HAVE_REFERENCE
-//    return PredictDeepNet<DNN::TReference<ScalarImpl_t> >(firstEvt, lastEvt, batchSize, logProgress);
-// #else
-   return std::vector<Double_t>(nEvents,TMath::QuietNaN());
-//#endif
-
+   Log() << kERROR << "STANDARD architecture  is deprecated for MethodDL ! " << Endl << Endl;
```

Comment:
That doesn't look like deprecation - it's now unavailable, right? "Deprecation" means "will be removed in a future version", where here it's just gone without prior warning. Not really nice towards our users, unless it's unused or broken?
It'd be also good to state an alternative for people to use. Or even better, to use a fall-back.